src/test/java/com/palantir/docker/compose/service/ServiceClusterIntegrationTest.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ */
+
+package com.palantir.docker.compose.service;
+
+import com.palantir.docker.compose.DockerComposition;
+import com.palantir.docker.compose.connection.waiting.HealthChecks;
+import org.junit.Rule;
+import org.junit.Test;
+
+import java.io.IOException;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+
+public class ServiceClusterIntegrationTest {
+
+    private final DockerService postgresFromComposeFileSnippet = DockerService.fromDockerCompositionFile("src/test/resources/postgres-service.yaml")
+                                                                              .withHealthCheck("postgres", HealthChecks.toHaveAllPortsOpen());
+    private final DockerService postgresDefinedInline = DockerService.fromImage("kiasaki/alpine-postgres", "inlinePostgres")
+                                                                     .withPortMapping(5432)
+                                                                     .withHealthCheck(HealthChecks.toHaveAllPortsOpen());
+
+    private final ServiceCluster services = ServiceCluster.of(postgresFromComposeFileSnippet, postgresDefinedInline);
+
+    @Rule
+    public DockerComposition composition = DockerComposition.fromServiceCluster(services)
```

Comment:
I should've read the tests first. Now I know what's going on.

I think this should be a more representative example of how we want people to use this, namely:

`DockerComposition.of(fileBackedPostgres, inlinePostgres)`